src/app/components/Brand/index.jsx

```diff
@@ -34,24 +40,36 @@ const VisuallyHiddenText = styled.span`
   top: 0;
 `;
 
+const StyledSpan = styled.span`
+  display: inline-block;
+  padding-bottom: ${GEL_SPACING_TRPL};
```

Comment:
Given that we have fixed values for the banner height and the margin-top of the BrandSvg, should this padding not also be a fixed value, rather than this `GEL_SPACING_TRPL`, which is currently 12px, but could increase when a page's font-size is increased by the user's browser settings. 

I suggest viewing this brand banner in the browser, increasing the font size to 200% or 300% to see if the branding looks correct in this case.

Reply:
Very good point! I've confirmed with UX and we think the banner height and SVG should grow to relative size as the font-size changes. This requires changing all the `px` values to be converted into `rem`'s. 

@greenc05 could you confirm this is the expected behaviour? 
EG: When a user increases the base font-size to 300% I would expect to see the: 
- Banner height increase from 80px to 240px
- SVG height and width increase from 24px and 167.95 to 72px and 503.85px retrospectively 
- All margin and padding spacings to increase by 3 times EG: 8 -> 24; 16 -> 48 

See screenshots:

**Font-size 100%**
![image](https://user-images.githubusercontent.com/7791726/44338667-89cfe700-a476-11e8-8f3f-a503776b704e.png)

**Font-size 300%**
![image](https://user-images.githubusercontent.com/7791726/44338691-a835e280-a476-11e8-8b28-27fcb0bc8587.png)